src/Campaigns/Controllers/CampaignRequestController.php

```diff
@@ -147,6 +146,21 @@ public function updateCampaign(WP_REST_Request $request)
         return new WP_REST_Response($campaign->toArray());
     }
 
+    /**
+     * @unreleased
+     *
+     * @throws Exception
+     */
+    public function mergeCampaigns(WP_REST_Request $request): WP_REST_Response
+    {
+        $destinationCampaign = Campaign::find($request->get_param('id'));
+        $campaignsToMerge = Campaign::query()->whereIn('id', $request->get_param('campaignsToMergeIds'))->getAll();
+
+        $campaignsMerged = $destinationCampaign->merge(...$campaignsToMerge);
+
+        return new WP_REST_Response($campaignsMerged);
+    }
```

Comment:
@alaca Not sure if this method is being used in any place. I think the `MergeCampaigns` class is still being used to merge the campaigns.

Reply:
@glaubersilva not sure why that is still there, I remember removing it.